app/src/main/res/layout/activity_main_content.xml

```diff
@@ -56,11 +56,30 @@
         </RelativeLayout>
     </android.support.design.widget.CoordinatorLayout>
 
-    <fragment
-        android:id="@+id/fragment_messages_container"
-        android:name="com.teamagam.gimelgimel.app.message.view.MessagesContainerFragment"
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        tools:layout="@layout/fragment_messages_container" />
+        android:orientation="vertical"
+      >
+
+        <com.astuetz.PagerSlidingTabStrip
+            android:id="@+id/tabs"
```

Comment:
The identifier should be more informative in such a way that will tell the developer which tabs exactly this id is referring to (main_activity_bottom_panel_tabs perhaps?)